algrtm/src/components/CustomSolutionsSection.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import CustomLink from "./shared/CustomLink";
+
+const CustomSolutionsSection = () => {
+  return (
+    <section className="flex mt-[3.945rem] w-full h-[25.5625rem] relative">
```

Comment:
We should add `overflow-hidden` to this section because the letter A goes beyond the page boundary, so we have scroll at the bottom:
![image](https://github.com/Nospoko/algrtm/assets/26410548/f44f8895-9c25-4ae0-a5b6-1e021452c98c)